EasyPost/Utilities/Enumeration.cs

```diff
@@ -0,0 +1,46 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+
+namespace EasyPost.Utilities
+{
+    public abstract class Enumeration : IComparable
```

Comment:
We definitely need tests for this

Reply:
What would a test for an enum look like?